utils/manifest-utils/resources/test/data.json.base64

```diff
@@ -0,0 +1 @@
+ewogICAgImZvbyI6ICJiYXIiCn0K
```

Comment:
[NITPICK] Consider writing here json and using `base64` crate directly in test

Reply:
I left it this way because it is just a fake data like { "foo": "bar" }, and this code doesn't care about its content. I bothered to make it an actual json because base64 encoded JSON is what what will be passed to this function, but I did not wanted to commit full manifest.json because it is large and I would need to prepare some fake content of it.